Automattic/jetpack
# Instant Search: add image alt text from API

Fixes #19273.

#### Changes proposed in this Pull Request:
The Instant Search API now provides `image.alt_text` as a field we can request.

This PR adds the new field and switches to use it for alt text in the expanded and product result formats.

I've also reduced the fields fetched for the minimal result format (image fields are not needed).

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions
1. On a site with Jetpack Instant Search enabled, create a post with an image. 
2. Set alt text for the image and ensure that the alt text appears on images in search results too (`/?s=`).

<img width="537" alt="Screen Shot 2022-01-11 at 15 57 40" src="https://user-images.githubusercontent.com/17325/148878738-debadc1c-4aa1-4f68-abe8-bbbad4e859b6.png">
